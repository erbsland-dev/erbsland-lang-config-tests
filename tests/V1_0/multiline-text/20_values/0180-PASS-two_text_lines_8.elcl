# Erbsland Configuration Language Test File
[multi_line_text]
value 34:
    """
    First Line

        Second Line
    """
