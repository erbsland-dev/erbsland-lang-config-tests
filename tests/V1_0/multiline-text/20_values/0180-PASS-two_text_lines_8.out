multi_line_text = SectionWithNames()
multi_line_text.value_34 = Text("First Line\u{a}\u{a}    Second Line")
